2020/day10/solution.py

```diff
@@ -0,0 +1,28 @@
+from collections import Counter
+from typing import Iterable
+
+def compute_gaps(numbers : Iterable[int]) -> Counter:
+    current = 0
+    gaps = Counter()
+    for num in numbers:
+        gaps[num - current] += 1
+        current = num
+    # add one more gap 3 for the final jump
+    gaps[3] += 1
+    return gaps
+
+def count_arrangements(numbers : Iterable[int]) -> int:
+    target = max(numbers)
```

Comment:
Do you need to include up the final case with the last number (3 higher than the last input)? If so, might be good to move the logic to add that number down to after parsing & below calling `compute_gaps` so both `compute_gaps` and `count_arrangements` get it for free.
If not, this is reasonable.

Reply:
I don't really because the last jump is +3 and that will only ever be the same as the number of ways to get to the max.  Since the i-1, i-2 will both be 0.